views/home.njk

```diff
@@ -117,7 +117,7 @@
         <div class="usa-width-two-thirds description">
           <h2>Launch your site in minutes.</h2>
           <p class="copy">
-            Is your code already on GitHub? Federalist can securely deploy a static website from your repository in minutes. Federalist also deploys site previews from GitHub branches, allowing your team to easily prototype site updates.
+            Federalist uses a [public code hosting service](https://github.com/) to load and store code for your site. If your code is already on GitHub, Federalist can securely deploy a static website from your repository in minutes. 
```

Comment:
We don't reference GitHub by name in the first sentence, but do in the second. Does that make sense?